payments-core/src/main/java/com/stripe/android/paymentsheet/model/StripeIntentValidator.kt

```diff
@@ -7,7 +7,7 @@ import com.stripe.android.model.StripeIntent
 /**
  * Validator for [PaymentIntent] or [SetupIntent] instances used in PaymentSheet.
  */
-internal class StripeIntentValidator {
+class StripeIntentValidator {
```

Comment:
This is the only additional class that needs to be exposed, and it could probably be injected into the PaymentSheetPaymentController.